src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -58,7 +58,7 @@
 
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
                         kube_config=None, kube_context=None, no_wait=False, tags=None, distribution='auto', infrastructure='auto',
-                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="600"):
+                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="600", enable_namespace_resource=False):
```

Comment:
Do we plan on modelling this as a feature which can be turned on/off using ARM based enable/disable?

Reply:
No, in the future this will be enabled by default. This is only for the private preview.